schemas/paths/event-search.yaml

```
@@ -94,7 +94,107 @@ get:
+    - name: frida
+      in: query
+      schema:
+        type: boolean
+      description: |
+        Filter events by Frida instrumentation detection result.  
+        > Note: When using this parameter, only events with the `products.frida.data.result` property explicitly set to `true` or `false` are returned. Events with undefined `products.frida` are left out of the response.
+    - name: factory_reset
+      in: query
+      schema:
+        type: boolean
+      description: |
+        Filter events by factory reset detection result.  
+        > Note: When using this parameter, only events with the `products.factoryReset.data.result` property explicitly set to `true` or `false` are returned. Events with undefined `products.factoryReset` are left out of the response.
+    - name: cloned_app
+      in: query
+      schema:
+        type: boolean
+      description: |
+        Filter events by cloned app detection result.  
+        > Note: When using this parameter, only events with the `products.clonedApp.data.result` property explicitly set to `true` or `false` are returned. Events with undefined `products.clonedApp` are left out of the response.
+    - name: emulator
+      in: query
+      schema:
+        type: boolean
+      description: |
+        Filter events by emulator usage result.  
+        > Note: When using this parameter, only events with the `products.emulator.data.result` property explicitly set to `true` or `false` are returned. Events with undefined `products.emulator` are left out of the response.
+    - name: root_apps
+      in: query
+      schema:
+        type: boolean
+      description: |
+        Filter events by root apps detection result.  
+        > Note: When using this parameter, only events with the `products.rootApps.data.result` property explicitly set to `true` or `false` are returned. Events with undefined `products.rootApps` are left out of the response.
+    - name: vpn_confidence
+      in: query
+      schema:
+        type: string
+        enum: [HIGH, MEDIUM, LOW]
+      description: |
+        Filter events by VPN confidence level result.  
+        > Note: When using this parameter, only events with the `products.vpn.data.result.confidence` property explicitly set to a valid value are returned. Events with undefined `products.vpn` are left out of the response.
+    - name: min_suspect_score
+      in: query
+      schema:
+        type: number
+        format: float
+      description: |
+        Filter events by suspect score result using a minimum threshold.  
+        > Note: When using this parameter, only events where the `products.suspectScore.data.result` property is explicitly set to a valid threshold and exceeds it are returned. Events with undefined `products.suspectScore` are left out of the response.
```

Tweaked phrasing for clarity, the event value is not a "threshold", the user-provided value is

```suggestion
        > Note: When using this parameter, only events where the `products.suspectScore.data.result` property is explicitly set to a value exceeding your threshold are returned. Events with undefined `products.suspectScore` are left out of the response.
```